### PR TITLE
Remove common from unlocalized actions

### PR DESCRIPTION
**Pull request by Albertlarsen**
## Description
Removes 'common' in front of unlocalized actions

## Related Issue(s)
- #598 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx [hunk 1]
```diff
@@ -61,19 +70,35 @@ export const RightsActionBarContent = ({
...
+                  <Chip.Toggle
+                    checkmark
+                    selected={right.checked}
+                    onClick={() => {
+                      toggleRight(serviceIdentifier, right.action);
+                    }}
+                  >
+                    {right.action}
+                  </Chip.Toggle>
+                </div>
```
Comment (allinox) [inline src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx:99]: I suggest setting the text to be displayed in a variable first.
Then you won't need the duplicate code of having the Chip-definitions twice
```suggestion
            const actionText = Object.values(LocalizedAction).includes(right.action as LocalizedAction) ? t(`common.${right.action}`) : right.action;
              return (
                <div key={index}>
                  <Chip.Toggle
                    checkmark
                    selected={right.checked}
                    onClick={() => {
                      toggleRight(serviceIdentifier, right.action);
                    }}
                  >
                    {actionText}
                  </Chip.Toggle>
                </div>
              );
```

File: src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx [hunk 2]
```diff
@@ -98,13 +123,21 @@ export const RightsActionBarContent = ({
...
-            .filter((r: Right) => r.delegable === false)
-            .map((r: Right, index: number) => {
-              return (
-                <div key={index}>
-                  <Chip.Toggle>{t(`common.${r.action}`)}</Chip.Toggle>
-                </div>
-              );
+            .filter((right: Right) => right.delegable === false)
+            .map((right: Right, index: number) => {
+              if (Object.values(LocalizedAction).includes(right.action as LocalizedAction)) {
```
Comment (allinox) [inline src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx:128]: Same goes here

**Review by allinox — COMMENTED** [review]
Great that you fixed this so quickly! 😊

File: src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx (hunk 2 above)
Comment (Albertlarsen) [reply src/features/singleRight/delegate/ChooseRightsPage/RightsActionBarContent/RightsActionBarContent.tsx:128]: Fixed

**Review by allinox — APPROVED** [review]
KpopApprovedGIF

**Pull request merged**